extension/vsc-js/README.md

```diff
@@ -0,0 +1,71 @@
+# howdoi README
+
+Are you a hack programmer? Do you find yourself constantly Googling for how to do basic programming tasks?
+
+Suppose you want to know how to format a date in bash. Why open your browser and read through blogs (risking major distraction) when you can simply stay in the code editor and ask howdoi:
+
+![Gif of howdoi commands within code editor](http://g.recordit.co/3795DyyMm1.gif)
+
+## Features
+
+The howdoi extension enables users to ask howdoi within their own code editor. 
+
+For usage within the code editor:
+
+#1 Write down your question in the code editor commented out with a single line comment
+
+![Gif of `\\ howdoi print python trace`](http://g.recordit.co/uPL4KqWvTV.gif)
+
+#2 Highlight #1
+
+![Gif of `\\ howdoi print python trace` highlight](http://g.recordit.co/ladhB94tBZ.gif)
+
+
+#3 Open the command palette using `cmd/ctrl + shift + P`
+
+#4 Run howdoi in command palette
+
+#5 Choose an option
+
+#6 The answer will be display in the code editor along with the question and link.
+
+## Requirements
+
+Install howdoi on your machine before using the howdoi extension.
+
+### Installation
```

Comment:
Will this README appear on the extension's page? Perhaps link back to the main howdoi github page for installation instructions since those might change from time to time.

Reply:
yes this will appear on the extensions's page. Will link it back to the main howdoi GitHub page for installation instructions.